Blish HUD/Controls/FlowPanel.cs

```diff
@@ -152,19 +200,50 @@ public class FlowPanel : Panel {
             }
         }
 
-        private void ReflowChildLayoutTopToBottom(IEnumerable<Control> allChildren) {
+        private void ReflowChildLayoutRightToLeft(IEnumerable<Control> allChildren)
+        {
+            float outerPadX = _padLeftBeforeControl ? _controlPadding.X : _outerControlPadding.X;
+            float outerPadY = _padTopBeforeControl ? _controlPadding.Y : _outerControlPadding.Y;
+
+            float nextBottom = outerPadY;
+            float currentBottom = outerPadY;
+            float lastLeft = this.Width - outerPadX;
+
+            foreach (var child in allChildren.Where(c => c.Visible))
+            {
+                // Need to flow over to the next row
+                if (outerPadX > lastLeft - child.Width)
+                {
+                    currentBottom = nextBottom + _controlPadding.Y;
+                    lastLeft = this.Width - outerPadX;
+                }
+
+                child.Location = new Point((int)(lastLeft - child.Width), (int)currentBottom);
+
+                lastLeft = child.Left - _controlPadding.X;
+                Debug.WriteLine(lastLeft - child.Width);
```

Comment:
You probably forgot this Debug statement in here

Reply:
I have fixed the formatting and removed the Debug statement.